app/helpers/concerns/foreman_salt/layout_helper_extensions.rb

```diff
@@ -1,20 +0,0 @@
-module ForemanSalt
-  module LayoutHelperExtensions
-    extend ActiveSupport::Concern
-
-    included do
-      alias_method_chain :authorized_associations_permission_name, :salt
```

Comment:
Damn I am just trying out salt for my LinuxDAYS 15 talk for the first time and it does not work.